### PR TITLE
Add the release blog for v0.88.0 and the Container Logs pages

### PR DESCRIPTION
**Pull request by humivo**
Add the release blog post and download link for ADOT EKS Add-on v0.88.0. Also add the Container Logs page.

### CODE REVIEW

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: 'Container Logs Collector Configuration'
+description:
+    This page introduces Collector configuration for Container Logs for the ADOT Collector.
+path: '/docs/getting-started/adot-eks-add-on/config-container-logs'
+---
+The Container Logs collector configuration launches a preconfigured OpenTelemetryCollector custom resource
```
Comment (mhausenblas) [inline src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx:7]: OpenTelemetryCollector -> OpenTelemetry Collector

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
+---
+title: 'Container Logs Collector Configuration'
+description:
+    This page introduces Collector configuration for Container Logs for the ADOT Collector.
+path: '/docs/getting-started/adot-eks-add-on/config-container-logs'
+---
+The Container Logs collector configuration launches a preconfigured OpenTelemetryCollector custom resource 
+to tail all Kubernetes pod logs with the `filelog` receiver. By opting into the
```
Comment (mhausenblas) [inline src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx:8]: link the receiver to https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+title: 'Container Logs Collector Configuration'
+description:
+    This page introduces Collector configuration for Container Logs for the ADOT Collector.
+path: '/docs/getting-started/adot-eks-add-on/config-container-logs'
+---
+The Container Logs collector configuration launches a preconfigured OpenTelemetryCollector custom resource 
+to tail all Kubernetes pod logs with the `filelog` receiver. By opting into the 
+[available pipeline](/docs/getting-started/adot-eks-add-on/config-container-logs/#container-logs-pipelines)
+you can control where the collected logs are sent. In this preconfigured custom resource, 
+the collector will run as a daemonset in order to capture all logs on each node.
```
Comment (mhausenblas) [inline src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx:11]: daemonset -> [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+| Service Account Name       | Namespace | IAM Policies |
+| -------------------------- | --------- | ------------ |
+| <nobr>adot-col-container-logs</nobr> | <nobr>opentelemetry-operator-system</nobr> | CloudWatchAgentServerPolicy |
+**Note:** Only attach the minimum set of policies necessary for your advanced configuration.
+
+The [IAM Roles for Service Accounts documentation (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.htmlhttps://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) contains instructions for creating the IAM
+role. The following examples will use [eksctl](https://eksctl.io/usage/iamserviceaccounts/https://eksctl.io/usage/iamserviceaccounts/) to achieve this. 
+
+To create this IAM role, run the following command:
```
Comment (mhausenblas) [inline src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx:28]: For example to create this IAM role using `eksctl` and annotate the service account with it, run the following command:

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        eks.amazonaws.com/role-arn: <iam_role_arn>
+```
+
+## Container Logs pipelines
+The following pipelines are available for the Container Logs preconfigured custom resource.
+Pipelines can be enabled by setting their `enabled` field to `true`. 
+
+### Logs
+
+- #### cloudwatchLogs
```
Comment (mhausenblas) [inline src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx:56]: #### Send to Amazon CloudWatch

**Review by mhausenblas — CHANGES_REQUESTED** [review]
LGTM!

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx (hunk 1 above)
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx]: Changed these occurrences.

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx (hunk 2 above)
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx]: Linked to receiver now.

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx (hunk 3 above)
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx]: Linked and changed capitalization.

File: src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx (hunk 4 above)
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/config-container-logs.mdx:28]: I feel like this suggestion is repeating the sentence we say above: `The following examples will use [eksctl](https://eksctl.io/usage/iamserviceaccounts/https://eksctl.io/usage/iamserviceaccounts/) to achieve this.`

**Review by mhausenblas — APPROVED** [review]

**Pull request merged**